.github/workflows/gh-pages.yml

```diff
@@ -3,30 +3,29 @@ name: github pages
 on:
   push:
     branches:
-    - deploy
+      - deploy
 
 jobs:
   build-deploy:
     runs-on: ubuntu-18.04
     steps:
-    - uses: actions/checkout@master
-      # with:
-      #   submodules: true
+      - uses: actions/checkout@master
+        # with:
+        #   submodules: true
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+          registry-url: "https://registry.npmjs.org"
 
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '10.x'
-        registry-url: 'https://registry.npmjs.org'
+      - name: Install
+        run: yarn
 
-    - name: Install
-      run: npm install --dev
+      - name: Build
+        run: yarn run build # TODO: reenable source maps as soon as it is possible - useful for debugging.
```

Comment:
I need to remove the comment about re-enabling source maps... (because I've eneabled them)

Also this uses yarn now